apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/plugins/editor-template-classes/index.js

```diff
@@ -30,7 +30,7 @@ const EditorTemplateClasses = withSelect( ( select ) => {
 	return { templateClasses };
 } )( ( { templateClasses } ) => {
 	const blockListInception = setInterval( () => {
-		const blockListParent = document.querySelector( '.block-editor__typewriter > div' );
```

Comment:
The TypeWriter component was [switched to use hooks](https://github.com/WordPress/gutenberg/pull/26986) which eliminated this class.